### PR TITLE
Added Images folder incuding raw image png, image data as pickle file, and module to get from one to the other.

### PR DESCRIPTION
**Pull request by iglesias-cardinale**
## Task Description
  - Take 1 picture of different periodic structures and add a size marker in the image (e.g. a yard stick) and store it in a pandas dataframe that is exported to a pickle in the repository (1 member due Monday 2024-11-11 midnight)

## Linting
  - Passes pylint with 10/10

## Contents
  - Images Folder
        -> ```image_to_pickle.py```  module the takes in a png in the Images folder, converts the image data to a pandas dataframe and converts that dataframe to a pickle file which is then saved in the Images folder. The actual data is stored in the pandas dataframe with a single row and "image_data" column that holds the entire image (3d) array. It's 3d since this is an rgb image. I can add some documentation on this if necessary.
         -> ```img_data.pkl``` pickle file created by ```image_to_pickle.py```
         -> ```labroom_floor.png``` png image of the floor of the undergraduate labs on the 3rd floor of Fronczak. Periodic in two dimensions and contains two metersticks at $90^\circ$ for scale. I'm not sure if this is the type of image that was intended, so I'm submitting my PR early so that we can discuss.

## Labels and Milestones
  - ALL pull requests must have an attached label and Milestone before a reviewer can be assigned.

## Reviewer
  - @avgagliardo or @ojha-aditya might be good for this since they're doing a similar task but using blender.

### CODE REVIEW

**Comment by avgagliardo** [comment]
I can review this tonight.

**Comment by kylemasc917** [comment]
@iglesias-cardinale Make sure you assign the appropriate milestone for your request. If it was not made already, please do so.

File: Images/image_to_pickle.py
Comment (avgagliardo) [inline Images/image_to_pickle.py:1]: File is well formed, documented, and lints

File: Images/img_data.pkl
Comment (avgagliardo) [inline Images/img_data.pkl:1]: pickle was imported without issue

**Review by avgagliardo — APPROVED** [review]
files reviewed and appear to be well-formed and properly documented.

pickle imported for testing.

approving for merge

**Comment by avgagliardo** [comment]
only thing left out is to maybe add a milestone

**Comment by iglesias-cardinale** [comment]
> pickle imported for testing.

unimaginably funny sentence out of context

**Comment by kylemasc917** [comment]
Ok everything looks in order Ill be merging this now.

**Pull request merged**